frontend/src/views/web/profile.html

```diff
@@ -1,24 +1,29 @@
-<div class="profile-sidebar z-depth-3">
-    <slide-profile></slide-profile>
-    <div class="ev-md-container center">
-        <img ng-src="{{profile.imgUrl}}" height="80" width="auto">
-        <div>{{profile.user.username}}</div>
-        <div class="text-light-gray w-300">{{profile.user.email}}</div>
-        <ul>
-            <li>
-                <span class="text-light-gray fs-12">First Name<br> </span> {{profile.user.first_name}}
-            </li>
-            <li>
-                <span class="text-light-gray fs-12">Last Name<br></span> {{profile.user.last_name}}
-            </li>
-            <li>
-                <span class="text-light-gray fs-12">Affiliated To<br></span> -
-            </li>
-        </ul>
-<!--         <li>
+<!-- <slide-profile></slide-profile> -->
+<div class="ev-md-container center">
+    <img src="dist/images/pro-pic.png" height="80" width="auto">
```

Comment:
## Proposed approach for Image size
- Make a mixing in `main.scss` name as `ImageSize` with two parameters `width` and `height`
- Give all the images a class.
- Call the mixin in the class with the arguments stating `width` and `height`